### ISSUE TITLE
[2022 spring] DEEP ANOMALY DETECTION WITH OUTLIER EXPOSURE (20205676)

### ISSUE THREAD

**Issue by Alex-C137E** [awesome-davian/awesome-reviews-kaist#510]
Good review about an interesting paper. Some part may lack a bit of detail, especially the related work where you talk about idea that are used after but don't take the time to introduced, so it makes your review a bit hard to follow for those you are not already familiar with the subject.

You should maybe also try to improve the overall organization of the text/images so it's a bit more pleasant to read.

Except that it's good.

**Comment by Joen0818** [comment]
Review(paper) show the importance of outlier data and anormaly detection. However, the readability of the review is poor due to minor problems such as paragraph spacing and figure size. I think it would be a better review if you could improve that part.
